src/cms/views/linkcheck/linkcheck.py

```diff
@@ -0,0 +1,106 @@
+from django.db.models import Q, OuterRef, Subquery
+from django.urls import reverse
+from django.views.generic import ListView
+from django.views.generic.base import RedirectView
+
+from linkcheck.models import Link
+from backend.settings import PER_PAGE
+
+from ...models.pages.page_translation import PageTranslation
+from ...models.events.event_translation import EventTranslation
+from ...models.pois.poi_translation import POITranslation
+
+
+# pylint: disable=too-many-ancestors
+class LinkListView(ListView):
+    """
+    View for retrieving a list of links grouped by their state
+    """
+
+    template_name = "linkcheck/links_by_filter.html"
+    context_object_name = "filtered_links"
+    paginate_by = PER_PAGE
+    extra_context = {"current_menu_item": "linkcheck"}
+
+    def get_context_data(self, **kwargs):
+        """
+        Appends additional context like the captured variable in the url
+
+        :return: The context rendered in the template
+        :rtype: dict
+        """
+        context = super().get_context_data(**kwargs)
+        context.update(
+            {
+                **self.kwargs,
+                "current_menu_item": "linkcheck",
+            }
+        )
+        return context
+
+    def get_queryset(self):
+        """
+        First filters all links by region
+        Next filters by links state and counts the number of links per group
+        Finally annotates queryset by the content_type title
+
+        :return: The QuerySet of the filtered links
+        :rtype: ~django.db.models.query.QuerySet
+        """
+        region_slug = self.kwargs.get("region_slug")
+        link_filter = self.kwargs.get("link_filter")
+        qset = Link.objects.filter(
+            Q(page_translations__page__region__slug=region_slug)
+            | Q(event_translations__event__region__slug=region_slug)
+            | Q(poi_translations__poi__region__slug=region_slug)
+        )
+        qset = qset.order_by("-url__last_checked")
+        valid_links = qset.filter(ignore=False, url__status__exact=True)
+        unchecked_links = qset.filter(ignore=False, url__last_checked__exact=None)
+        ignored_links = qset.filter(ignore=True)
+        invalid_links = qset.filter(ignore=False, url__status__exact=False)
+        self.kwargs.update(
+            {
```

Comment:
```suggestion
        self.extra_context.update(
            {
                "link_filter": link_filter,
```
You can directly write to `extra_context`, you just have to pass the additional variable `link_filter`...